src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/HopTransactionStep.tsx

```diff
@@ -155,20 +160,54 @@ export const HopTransactionStep = ({
 
   const { isFetching, data: tradeQuoteQueryData } = useGetTradeQuotes()
 
+  const feeAsset = useAppSelector(state =>
+    selectFeeAssetByChainId(state, tradeQuoteStep?.sellAsset.chainId ?? ''),
+  )
+  const feeAssetBalanceFilter = useMemo(
+    () => ({ assetId: feeAsset?.assetId ?? '', accountId: sellAssetAccountId ?? '' }),
+    [feeAsset?.assetId, sellAssetAccountId],
+  )
+  const feeAssetBalance = useAppSelector(state =>
+    selectPortfolioCryptoBalanceBaseUnitByFilter(state, feeAssetBalanceFilter),
+  )
```

Comment:
May as well try to suggest it

```suggestion
  const feeAssetBalance = useSelectorWithArgs(selectPortfolioCryptoBalanceBaseUnitByFilter, feeAssetBalanceFilter)
```

Reply:
hell yeah 2751c96d3a